homework3/EVPurchaseProgram2/src/main/java/ro/siit/java10/evp/CarFilter.java

```diff
@@ -0,0 +1,59 @@
+package ro.siit.java10.evp;
+
+import java.util.ArrayList;
+
```

Comment:
java docs are missing. 
I would suggest you to read Java Code Convention - Oracle